announcement.md

```diff
@@ -0,0 +1,3 @@
+## Abstract
```

Comment:
I'd be careful about statements about Python being less performant. This may or may not be true but is almost certain to upset the (Python loving) audience.
Maybe add "often not as performant" instead.

Reply:
If we have some particular audience in mind I would mention that. Eg we are not covering machine learning packages but more low level GPU programming that is applicable to generic programming (eg numba, pycuda). cudf is mostly useful for data preparation since it handles dataframes like pandas does but not geared towards (lots of) numerical computing (though it likely can be done).